learning_journal/scripts/initializedb.py

```diff
@@ -50,7 +50,7 @@ def main(argv=sys.argv):
     options = parse_vars(argv[2:])
     setup_logging(config_uri)
     settings = get_appsettings(config_uri, options=options)
-    settings["sqlalchemy.url"] = os.environ['DATABASE_URL']
+    settings["sqlalchemy.url"] = os.environ.get('DATABASE_URL', 'sqlite:///:memory:')
```

Comment:
you probably don't want to have your database url default to in-memory if there is no env var specified...  Think about how this affects persistance.